### DISCUSSION TITLE
Why Hono is slow in web framework bentchmak ?

### DISCUSSION THREAD

**Discussion by rajeshkhetarpal** [honojs/hono#1192] [General]
https://web-frameworks-benchmark.netlify.app/result?f=hono,fastify

**Comment by yusukebe** [comment]
Already mentioned:

https://twitter.com/yusukebe/status/1668623884856471558

https://twitter.com/jarredsumner/status/1668864999052427264

**Reply by rajeshkhetarpal** [reply]
thanks
